qt/python/mantidqt/widgets/fitpropertybrowser/fitpropertybrowser.py

```diff
@@ -368,10 +377,14 @@ def fitting_done_slot(self, name):
         for new_line, old_line in zip(new_lines, original_lines):
             new_line.update_from(old_line)
 
-        handles, labels = axes.get_legend_handles_labels()
+        if plot_has_legend:
+            handles, labels = axes.get_legend_handles_labels()
```

Comment:
What does this function do, `handles` and `labels` do not seem to be used.

Reply:
I checked with the author and it was left in by mistake after resolving a merge conflict. Will remove 👍